src/api/app/jobs/project_status_calculator.rb

```diff
@@ -211,7 +212,7 @@ def calc_status(opts = {})
 
     links = Array.new
     # find links
-    mypackages.values.each do |package|
+    mypackages.each_value.each do |package|
       if package.project == @dbproj.name and package.links_to_id
```

Comment:
if you do .values, you create an array, that isn't necessary. Better to use an iterator...